src/app/modules/main/profile_section/privacy/view.nim

```diff
@@ -61,10 +61,8 @@ QtObject:
   proc getUrlUnfurlingMode(self: View): int {.slot.} =
     return self.delegate.urlUnfurlingMode()
   proc setUrlUnfurlingMode(self: View, value: int) {.slot.} =
-    if self.getUrlUnfurlingMode() == value:
-      return
```

Comment:
hmm, why not?

Reply:
Fixing a non-issue and introducing a real one.. Thanks for spotting it! Removed